packages/widgets/src/blocks/widget-group/index.php

```diff
@@ -8,6 +8,9 @@
 /**
  * Renders the 'core/widget-group' block.
  *
+ * @global array $wp_registered_sidebars
+ * @global array $_sidebar_being_rendered
```

Comment:
```suggestion
 * @global array      $wp_registered_sidebars
 * @global int|string $_sidebar_being_rendered
```

The type of `$_sidebar_being_rendered` should be int or string. This is because the argument value received [here](https://github.com/WordPress/gutenberg/blob/4bb17c47afb4741ebcfb54b1e498f27ed9831159/packages/widgets/src/blocks/widget-group/index.php#L62-L65) is assigned as is.